client/account-recovery/account-recovery-root/index.jsx

```diff
@@ -35,13 +41,44 @@ const getPageInfo = ( translate, step ) => {
 			trackerTitle: 'Account Recovery > Reset Password',
 			documentHeadTitle: translate( 'Reset Password ‹ Account Recovery' ),
 		},
-
+		[ STEPS.RESET_PASSWORD_EMAIL ]: {
+			trackerTitle: 'Account Recovery > Reset Password Email',
+			documentHeadTitle: translate( 'Reset Password < Email' ),
+		},
+		[ STEPS.RESET_PASSWORD_SMS ]: {
+			trackerTitle: 'Account Recovery > Reset Password Sms',
+			documentHeadTitle: translate( 'Reset Password ‹ SMS Message' ),
+		},
+		[ STEPS.RESET_PASSWORD_CONFIRM ]: {
+			trackerTitle: 'Account Recovery > New Password',
+			documentHeadTitle: translate( 'Reset Password < New Password' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **79** times:
`translate( 'Repeat New Password' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).